test/sourcepos.test

```diff
@@ -11,3 +11,21 @@ doc
       para (2:4:8-3:0:9)
         str (2:4:8-2:4:8) text="b"
 ```
+
+``` ap
+ - a
+ - b
+
+c
+.
+doc
+  bullet_list (1:2:1-4:0:10) tight=true style="-"
+    list_item (1:2:1-2:1:5)
+      para (1:4:3-2:0:4)
+        str (1:4:3-1:4:3) text="a"
+    list_item (2:2:6-3:0:9)
+      para (2:4:8-3:0:9)
```

Comment:
Where should this para ends? 4:0;10, 3:0:9 or 2:4:8?

Reply:
I'd say that if paragraph "a" ends at 2:0, then "b" should end at "3:0" for consistency.
Alternatively we could change things so that all paragraphs end at their last newline character or last text letter, but I think the minimal change would be the one above.